README.md

```diff
@@ -76,37 +75,86 @@ let dog = new Dog("Spot");
 dog.speak(); // Spot speaks! Spot says woof!
 ```
 In `Dog`s `constructor`, `super` is used as a `method`. Whereas, `Dog`'s `speak()`
-method used `super` as an `object`. When the `super` keyword is used as method, it
+method used `super` as an `object`. When the `super` keyword is used as a method, it
 calls the parent `class` `Pet` with the parameters passed to `Dog`. This is to ensure
 that `Dog` is an instance of `Pet`. When `super` is used as an `object` that refers to
-a `Pet` instance, it can calls methods of the parent `class` `Pet` explicitly.
+a `Pet` instance, it can call methods of the parent `class` `Pet` explicitly.
 
 ## Explain When `class` Inheritance Should Be Used in JavaScript
 
-JavaScript can be a bit confusing since it is a dynamic language that uses syntactical
+You probably won't use inheritance often, but as your code bases get larger, you are more
+likely to find a need for it. If you're creating a number of objects that have similar
+features, then creating a generic object type to contain all the shared functionality,
+and inheriting those features in more specialized objects can be convenient and useful.
+
+JavaScript can be a bit confusing since it is a dynamic language that uses syntactic
 sugar to leverage `class` implementations and inheritance similar to classic programming
 languages. However, JavaScript's inheritance model is quite powerful and fairly trivial
 to build on top of like when using a classic programming language model. As the popular
 phrase goes: "With power comes great responsibility." It can be tempting to overuse these
-features. When creating "child" `class`es, we defer to _subtyping_ relation principles
-called _(strong) behavioral subtyping_ in programming language theory. 
+features.
+
+When creating child `class`es, we defer to _subtyping_ relation principles
+called _(strong) behavioral subtyping_ in programming language theory. A parent-child 
```

Comment:
This too can possibly be its own subsection

'### Strong Behavioral Subtyping'

' _(strong) behavioral subtyping_ in programming language theory' I think needs to be unpacked.

definitely link PLT to: https://en.wikipedia.org/wiki/Programming_language_theory 

SB subtyping: https://en.wikipedia.org/wiki/Liskov_substitution_principle

Reply:
Maybe to intro this section, mention SOLID: https://en.wikipedia.org/wiki/SOLID

Solid is a set of design principles for OO programming. The L in solid is the subtyping thing. so to adhere to good design, we try to maintain  (strong) behavioral subtyping.